internal/cli/release_list.go

```diff
@@ -0,0 +1,391 @@
+package cli
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/waypoint/internal/version"
```

Comment:
This might be something you'll want to configure Goland to do, but typically we group package imports together like:

- stdlib pkgs
- external
- hashicorp specific

I know my `vim-go` sorts them automatically so there's probably some switch that golang has for it!

Reply:
Hm I copied this part straight from deployment_list.go so this could go on the list of hack week things to clean up